### PR TITLE
Meteor Balance patch 4

### PR DESCRIPTION
**Pull request by ZomgPonies**
- Delay between waves lowered to 5-10 minutes
- Forced maximum possible intensity to 25, after 5 waves. Original intensity of gamemode was 20(which is considered more intense in code).
- After 5 waves, only duration of waves will increase, not intensity.

### CODE REVIEW

**Comment by Iamgoofball** [comment]
>meteor
>balance

Godspeed and good luck

**Comment by ZomgPonies** [comment]
@Iamgoofball If you'd been paying attention you'd notice this is the 4th patch, and meteor is well on it's way to being balanced considering it's back in rotation.

File: code/game/gamemodes/meteor/meteor.dm [hunk 1]
```diff
@@ -24,10 +24,10 @@
 
 
 /datum/game_mode/meteor/proc/sendmeteors()
-	var/waveduration = world.time + rand(0,1000) + text2num("[wave]000")
-	var/waitduration = rand(3000,9000)
+	var/waveduration = world.time + rand(0,1000) + text2num("[wave]000" / 2)
```
Comment (Tastyfish) [inline code/game/gamemodes/meteor/meteor.dm]: @ZomgPonies, you divided a string by 2 and then passed the result as a string. might be why the mode doesn't work

Comment (Tastyfish) [inline code/game/gamemodes/meteor/meteor.dm]: Yeah making it 
```c
var/waveduration = world.time + rand(0,1000) + text2num("[wave]000") / 2
```
seems to make it work in a way that I assume is intended, I'll let you make the PR

**Pull request merged**